### PR TITLE
qtgui: Add eye sink widget

### PR DESCRIPTION
**Pull request by ChristopheSeguinot**
Fourth attempt to get a clean PR.

### CODE REVIEW

File: gr-qtgui/include/gnuradio/qtgui/eye_sink_c.h [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+ *
+ * This sink can plot messages that contain either uniform vectors
+ * of complex 32 values (pmt::is_c32vector) or PDUs where the data
+ * is a uniform vector of complex 32 values.
+ */
+class QTGUI_API eye_sink_c : virtual public gr::sync_block
+{
+public:
+    // gr::qtgui::eye_sink_c::sptr
+    typedef boost::shared_ptr<eye_sink_c> sptr;
```
Comment (drmpeg) [inline gr-qtgui/include/gnuradio/qtgui/eye_sink_c.h:59]: Needs to be std::shared_ptr

File: gr-qtgui/include/gnuradio/qtgui/eye_sink_f.h [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+ * This sink can plot messages that contain either uniform vectors
+ * of float 32 values (pmt::is_f32vector) or PDUs where the data
+ * is a uniform vector of float 32 values.
+ */
+
+class QTGUI_API eye_sink_f : virtual public gr::sync_block
+{
+public:
+    // gr::qtgui::eye_sink_f::sptr
+    typedef boost::shared_ptr<eye_sink_f> sptr;
```
Comment (drmpeg) [inline gr-qtgui/include/gnuradio/qtgui/eye_sink_f.h:59]: Needs to be std::shared_ptr

**Comment by drmpeg** [comment]
Your PR is failing the build because of the change from boost::shared_ptr to std::shared_ptr on the 3.9 branch.

This occurred in commit https://github.com/gnuradio/gnuradio/commit/7a9169fe8cca1cb378be0d0d403e03a338ffbfda

**Comment by mormj** [comment]
adding a couple more gr devs to approve so we can get this merged before it goes stale

**Comment by mormj** [comment]
@ChristopheSeguinot - do you have a CLA on file with us?  If so, I can go ahead and merge
https://wiki.gnuradio.org/index.php/Development#What.27s_this_Copyright_Assignment.3F

**Comment by ChristopheSeguinot** [comment]
Yes I've a CLA (Seguinot.1468222.GNU.RADIO_cla.pdf, dated 2020 February 6)

**Comment by mormj** [comment]
Great!  Merging now.

**Pull request merged**